hub/core/dataset.py

```diff
@@ -475,9 +564,11 @@ def __str__(self):
         if self.index.is_trivial():
             index_str = ""
 
-        return f"Dataset({path_str}{mode_str}{index_str}tensors={self.meta.tensors})"
+        return f"Dataset({path_str}{mode_str}{index_str}tensors={self.version_state['meta'].tensors})"
 
     __repr__ = __str__
+    tf = tensorflow
+    pt = pytorch
```

Comment:
This is @mccrearyd 's hactoberfest [issue](https://activeloop.atlassian.net/browse/AL-1149) :p Also, not sure about "pt" alias..

Reply:
ah, accidentally slipped in this. removing